features/google/src/components/awakeningLocationMinistory/awakeningLocationMinistory.jsx

```diff
@@ -0,0 +1,135 @@
+import "./awakeningLocationMinistory.css";
+import React, { useState } from "react";
+import SourceInfoButton from "../sourceInfoButton/sourceInfoButton.jsx";
+import i18n from "!silly-i18n";
+
+export const AwakeningLocationSummary = ({ dateData }) => {
+    const defaultDate = Object.entries(dateData).find(
+        (entry) => !!entry[1].firstActivity
+    )[0];
+    const [selectedDate, setSelectedDate] = useState(defaultDate);
+    const data = dateData[selectedDate];
+    const dateSelection = false;
+    const handleChangeDate = ({ target }) => {
+        const formattedDate = target.value
+            .split("-")
+            .reverse()
+            .map((e) => parseInt(e))
+            .join("-");
+        setSelectedDate(formattedDate);
+        console.log(formattedDate);
+    };
+    //could be used to make the text more human later on
+    // const week = [
+    //     "Sunday",
+    //     "Monday",
+    //     "Tuesday",
+    //     "Wednesday",
+    //     "Thursday",
+    //     "Friday",
+    //     "Saturday",
+    // ];
+    return (
```

Comment:
Did you clear this with design? Are there too many dates so it's best to select one? Couldn't we pre-select some dates for them, as in those dates that they were not at home or something like that? In general, picking up a random date or some date they already know and knowing they were at home, or where they knew they would be, does not seem like such a big deal, right?

Reply:
Hmm, I did clear it with design, and I think for the time being, the random date is alright. The details page offers a deeper dive anyways so if they are interested in checking out all the registered mornings, they can do that as well.

Reply:
What about simply selecting a random date for them?
I want to avoid temporary solutions that add to the UI, simply to be eliminated later on... 
My point is, think about them selecting three or four random dates, getting "you were at home", and that would be it. They would never return to the feature.